javav2/usecases/Creating_Spring_Redshift_ Rest/Readme.md

```diff
@@ -0,0 +1,1125 @@
+# Creating a React and Spring REST application that queries Amazon Redshift data
```

Comment:
It looks like the directory containing this file has a space in it, as well as underscores. Is that intentional? The space should probably be removed.